database/index.js

```diff
@@ -21,6 +21,7 @@ const photosAndComments = mongoose.model('photosAndComments', photosAndCommentsS
 
 module.exports.photosAndComments = photosAndComments;
 module.exports.db = db;
+module.exports.photosAndCommentsSchema = photosAndCommentsSchema;
```

Comment:
@aku2nnya Recommend re-factoring this to something like :
module.exports: {
photosAndComments,
db,
photosAndCommentsSchema,
}

--this would do the same thing you have here!

Reply:
Didn't know this was possible, thanks!